nuxt.config.js

```diff
@@ -117,4 +117,7 @@ export default {
       await addSearch(nuxt);
     },
   },
+  generate: {
+    fallback: true,
```

Comment:
what's this do?

Reply:
this was a leftover non-working workaround for the issue you helped me solve where the 404 page wasnt showing up. deleting this.